setup.py

```diff
@@ -101,4 +101,7 @@
             'ots-git-gpg-wrapper = otsclient.git_gpg_wrapper:main',
         ],
     },
+
+    # Install the ots-git-gpg-wrapper.sh-script
+    scripts = ["ots-git-gpg-wrapper.sh"],
```

Comment:
One issue with installing the script is it'll clutter up tab-completion with another program starting with ots. Is there a better place to put this? Under /usr/lib/ or something?

Reply:
See https://github.com/opentimestamps/opentimestamps-client/pull/121#discussion_r719199978

Not putting it into `PATH` forces one to hard-code an absolute path with `git config gpg.program`. For people synchronizing their `.gitconfig` for example this is annoying as the wrapper can be installed in different places on different machines.